.travis.yml

```diff
@@ -10,16 +11,19 @@ sudo: false
 script: mix test
 matrix:
   include:
-      otp_release: 21.0
-    - elixir: 1.7
-      otp_release: 19.3
-    - elixir: 1.7
-      otp_release: 20.3
-    - elixir: 1.7
-      otp_release: 21.0
-    - elixir: 1.8
-      otp_release: 20.3
-    - elixir: 1.8
-      otp_release: 21.0
-    - elixir: 1.8
-      otp_release: 22.0.1
+    - elixir: '1.7'
+      otp_release: '19.3'
+    - elixir: '1.7'
+      otp_release: '20.3'
+    - elixir: '1.7'
+      otp_release: '21.0'
+    - elixir: '1.8'
+      otp_release: '20.3'
+    - elixir: '1.8'
+      otp_release: '21.0'
+    - elixir: '1.8'
```

Comment:
do you want to keep these old versions in the matrix? how far back?

Reply:
Also, let me know if you'd prefer to switch to GitHub Actions instead.

Reply:
Hey nathany! thanks for starting this! Yes please I would love to move us to Github Actions!

Reply:
And happy to drop old versions. Happy to stay with the Elixir 1.15+ if that's possible